elasticreindexer/process/reindexer_test.go

```diff
@@ -60,7 +60,7 @@ func testCopyMappingNoOverwriteShouldCreate(t *testing.T) {
 
 	r, _ := newReindexer(sourceClient, destinationClient, []string{"index"})
 
```

Comment:
Shouldn`t we test also for skipMappings == true ?

Reply:
added

Reply:
You still have 1/4 possible scenarios to test: skipMappings == true && override == true. Does it make sense to have it tested too ?

Reply:
I think is enough only the part when `skipMapping` is true  because will early exits the function `copyMappingIfNecessary`